packages/cli/src/commands/new/new.ts

```diff
@@ -14,17 +14,66 @@
  * limitations under the License.
  */
 
-import os from 'os';
-import fs from 'fs-extra';
-import { join as joinPath } from 'path';
-import { OptionValues } from 'commander';
-import { FactoryRegistry } from '../../lib/new/FactoryRegistry';
-import { isMonoRepo } from '@backstage/cli-node';
-import { paths } from '../../lib/paths';
-import { assertError } from '@backstage/errors';
-import { Task } from '../../lib/tasks';
+import { createNewPackage } from '../../lib/new/createNewPackage';
 
-function parseOptions(optionStrings: string[]): Record<string, string> {
+type ArgOptions = {
+  option: string[];
+  select?: string;
+  skipInstall: boolean;
+  private?: boolean;
+  npmRegistry?: string;
+  scope?: string;
+  license?: string;
+  baseVersion?: string;
+};
+
+export default async (opts: ArgOptions) => {
+  const {
+    option: rawArgOptions,
+    select: preselectedTemplateId,
+    skipInstall,
+    scope,
+    private: isPrivate,
+    ...otherGlobals
+  } = opts;
+
+  const prefilledParams = parseParams(rawArgOptions);
+
+  let pluginInfix: string | undefined = undefined;
+  let packagePrefix: string | undefined = undefined;
+  if (scope) {
+    const backstagePrefix = scope.startsWith('backstage') ? '' : 'backstage-';
```

Comment:
should there be an `@` here or no?

Reply:
Nop the old scope was defined without `@`